### ISSUE TITLE
As a consumer, I want to see whether a school's outcome measures are above/about/below average from the search results and school profile page, so that I can evaluate a school without clicking on anything.

### ISSUE THREAD

**Issue by saintsoup52** [18F/college-choice#508]
Don’t bury the lead: The Above, About and Below Average should be above the fold on the search results and the profile page. Currently the only way for a user to evaluate the three key outcomes they see when they search for a school or open a school’s profile is the national average. However, when they open the drawers they get the much more useful “above, about and below” average information.

**Comment by LisaGee** [comment]
Good point, although the bar charts themselves are supposed to serve as those indicators.
But we're taking a diff crack at the display (actually was your original suggestion), but suspect it may not make it for launch. See #531
